src/screens/AccountTransactions/index.tsx

```diff
@@ -0,0 +1,223 @@
+  }
+
+  async function refreshBlockchainHeight() {
+    console.log('Retreiving blockchain height...');
+    const height = await accountsContext.getBlockchainHeight();
+    console.log('Blockchain Height', height);
+    setBlockchainHeight(height);
+  }
+
+  async function refreshAccount() {
+    const account = accountsContext.currentAccount;
+
+    const externalDescriptor = await new Descriptor()
+      .create(accountsContext.currentAccount.external_descriptor as string, Network.Testnet);
+    const internalDescriptor = await new Descriptor()
+      .create(accountsContext.currentAccount.internal_descriptor as string, Network.Testnet);
+
+    const wallet = await accountsContext.loadWalletFromDescriptor(externalDescriptor, internalDescriptor);
+    console.log('Syncing wallet...');
+
+    await accountsContext.syncWallet(wallet);
+    console.log('Completed wallet sync.');
+
+    await accountsContext.populateWalletData(wallet, account);
+    await accountsContext.storeAccount(account);
+  }
+
+  function txnSortAsc(txn1: Transaction, txn2: Transaction) {
+    const t1 = new Date(txn1.timestamp as Date);
+    const t2 = new Date(txn2.timestamp as Date);
+    return (t1?.getTime() || 0) - (t2?.getTime() || 0);
+  }
+
+  function txnSortDesc(txn1: Transaction, txn2: Transaction) {
+    const t1 = new Date(txn1.timestamp as Date);
+    const t2 = new Date(txn2.timestamp as Date);
+    return (t2?.getTime() || 0) - (t1?.getTime() || 0);
+  }
+
+  return (
+    <AccountsContext.Consumer>
+      {({currentAccount: account}) => (
+        <View style={styles.container}>
+          <BackgroundGradient orientation={'horizontal'}>
+            <View style={styles.header}>
+              <Sats sats={account?.summary?.balanceSats} satsStyle={styles.sats} satsLabelStyle={styles.satsLabel} usdStyle={styles.usd} usdLabelStyle={styles.usdLabel} />
+            </View>
+            <GradientSeparator />
+            <ActionBar />
+            <GradientSeparator />
+            <AccountSummaryTabs summary={account.summary}/>
+          </BackgroundGradient>
+          <View style={styles.transactionsHeaderContainer}>
+            <View style={styles.transactionsHeader}>
+              <TouchableOpacity
+                style={styles.action}
+                activeOpacity={0.7}
+                onPress={onRefresh}
+              >
+                <RefreshIcon width={18} height={18} />                
+              </TouchableOpacity>
+              { refreshing ?
+                <AppText style={[styles.transactionsHeaderText, styles.transactionsHeaderTextRefreshing]}>Updating Parent Account Activity...</AppText> :
+                <AppText style={styles.transactionsHeaderText}>Parent Account Activity</AppText>
+              }
+              <SortDirectionToggle
+                style={styles.action}
+                onDirectionChanged={(direction: SortDirection) => setSortDirection(direction)}
+              />
+            </View>
+          </View>
```

Comment:
Before I was thinking to not separate out the header and listing as components and leave them in AccountTransactionsScreen because just seemed like so much props passing, but after having done it I like it 👍